src/server/config.ts

```diff
@@ -261,7 +261,7 @@ export const SETTINGS_MANAGER = new SettingsManager(settingsStore, {
 if (PRINT_CONFIG) {
   var withComments = Boolean(parsedArgs["with-comments"]);
 
-  SETTINGS_MANAGER.getSettings({
+  SETTINGS_MANAGER.getFreshSettings({
```

Comment:
refreshSettings?

Reply:
`refreshSettings` sound like action with some side effect, but this function is still a getter. Honestly I don't like either option.